Report generation fails, if path to local maven repository contains spaces

**Describe the bug**
We recently switched from the report module to report-ng. Since then no report is generated when running inside a Jenkins-Pipeline which contains spaces in its name. This seems to be caused by an improperly escaped path to the report-ng jar file.

**To Reproduce**
Steps to reproduce the behavior:
1. Clone the testerra-skeleton repo
2. Create a directory with a space in it, e.g. `C:\Local Maven Repo`
3. Run maven inside the skeleton project with this directory as local repo: `mvn  -Dmaven.repo.local="C:\Local Maven Repo" test -PmySuite`

**Expected behavior**
A report to be present in the target directory.

**Actual behavior**
No report is present and the following stacktrace is logged:
```
2021-04-27 21:06:18,328 [main] ERROR eu.tsystems.mms.tic.testframework.listeners.CopyReportAppListener - Unable to copy app resource
java.lang.IllegalArgumentException: Illegal character in opaque part at index 49: jar:file:/var/lib/jenkins/workspace/Test-Pipeline Abnahme/.repository/eu/tsystems/mms/tic/testerra/report-ng/1-SNAPSHOT/report-ng-1-SNAPSHOT.jar
	at java.net.URI.create(URI.java:852)
	at eu.tsystems.mms.tic.testframework.listeners.CopyReportAppListener.getPathsFromResourceJAR(CopyReportAppListener.java:83)
	at eu.tsystems.mms.tic.testframework.listeners.CopyReportAppListener.onFinalizeExecution(CopyReportAppListener.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:87)
	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:144)
	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:72)
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:398)
	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:67)
	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:108)
	at com.google.common.eventbus.EventBus.post(EventBus.java:212)
	at eu.tsystems.mms.tic.testframework.report.TesterraListener.generateReport(TesterraListener.java:418)
	at org.testng.TestNG.generateReports(TestNG.java:1099)
	at org.testng.TestNG.run(TestNG.java:1042)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
Caused by: java.net.URISyntaxException: Illegal character in opaque part at index 49: jar:file:/var/lib/jenkins/workspace/Test-Pipeline Abnahme/.repository/eu/tsystems/mms/tic/testerra/report-ng/1-SNAPSHOT/report-ng-1-SNAPSHOT.jar
	at java.net.URI$Parser.fail(URI.java:2848)
	at java.net.URI$Parser.checkChars(URI.java:3021)
	at java.net.URI$Parser.parse(URI.java:3058)
	at java.net.URI.<init>(URI.java:588)
	at java.net.URI.create(URI.java:850)
	... 22 common frames omitted
```

**Environment(please complete the following information):**
 - OS: CentOS 7
 - Version 1-SNAPSHOT

**Additional Information:**
From my research this could probably be fixed by changing the line https://github.com/telekom/testerra/blob/4db3879d0e0c41d7a11d091811a97e78de999e56/report-ng/src/main/java/eu/tsystems/mms/tic/testframework/listeners/CopyReportAppListener.java#L78 from `getPath()` to `getRawPath()`, i.e. using the already escaped path component of the URI. But I was not yet able to confirm this myself, as I had some issues building the module.

--- a/report-ng/src/main/java/eu/tsystems/mms/tic/testframework/listeners/CopyReportAppListener.java
+++ b/report-ng/src/main/java/eu/tsystems/mms/tic/testframework/listeners/CopyReportAppListener.java
@@ -75,7 +75,7 @@ public class CopyReportAppListener implements FinalizeExecutionEvent.Listener, L
                 .getCodeSource()
                 .getLocation()
                 .toURI()
-                .getPath();
+                .getRawPath();
 
         List<Path> result;
 

